src/WCSimPMTObject.cc

```diff
@@ -220,6 +222,12 @@ G4float PMT20inch::GetmaxQE(){
   return maxQE;
 }
 
+G4float* PMT20inch::GetCollectionEfficiencyArray(){
+  static G4float CE[10] = { 100., 100., 100., 100., 100., 100., 100., 100., 100., 100.};//temporary value. It will be changed in future.
+  //  static G4float CE[10] = { 73., 73., 73., 73., 73., 73., 73., 73., 73., 73.};
+  return CE;
+}
+
```

Comment:
Remove lines 225-230. We should aim to make minimal changes and is no need to add GetCollectionEfficiencyArray(); to the 20" PMTs in this pull request.